pkgs/development/libraries/boost/generic.nix

```
@@ -222,6 +222,15 @@ stdenv.mkDerivation {
         extraPrefix = "libs/python/";
         hash = "sha256-0IHK55JSujYcwEVOuLkwOa/iPEkdAKQlwVWR42p/X2U=";
       })
+    ]
+    ++ lib.optional (lib.versionAtLeast version "1.87") [
```

This will most likely break with version 1.88. You could use
```suggestion
    ++ lib.optional ((builtins.compareVersions version "1.87") == 0) [
```